integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -28,6 +28,38 @@ class MoodleContentMetadataExporter(ContentMetadataExporter):
     LONG_STRING_LIMIT = 1700  # Actual maximum value we can support for any individual course
     SKIP_KEY_IF_NONE = True
 
+    def transform_shortname(self, content_metadata_item):
+        """
+        We're prefixing Title to the key to make shortname a little nicer in Moodle's UI.
+        But because we use key elsewhere, I mapped this to itself
+        so it doesn't override all "key" references
+        """
+        return '{} ({})'.format(
+            content_metadata_item.get('title'),
+            content_metadata_item.get('key')
+        )
+
+    def transform_title(self, content_metadata_item):
+        """
+        Returns the course title with all organizations (partners) appended in parantheses
+        """
+        formatted_orgs = []
+        for org in content_metadata_item.get('organizations', ''):
```

Comment:
good old python iteration- stylistically, maybe an empty list instead of string? But functionally doesn't matter and it's so minute you could just not bother 🙃